breadth_first_search/BreadthFirstSearch.py

```diff
@@ -0,0 +1,31 @@
+import collections
+
+def bfs(graph, root):
+    returnprint = 'BFS Order: '
+    seen, queue = set([root]), collections.deque([root])
+    while queue:
+        vertex = queue.popleft()
+        returnprint += str(vertex) + '  '
+        for node in graph[vertex]:
+            if node not in seen:
+                seen.add(node)
+                queue.append(node)
+    print(returnprint)
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpisevponc/breadth_first_search/BreadthFirstSearch.py
+++ b/tmp/tmpisevponc/breadth_first_search/BreadthFirstSearch.py
@@ -11,6 +11,7 @@
                 seen.add(node)
                 queue.append(node)
     print(returnprint)
+
 
 def visit(n):
     print(n)
```